Birdee/Parser.cpp

```diff
@@ -1251,15 +1251,17 @@ BD_CORE_API bool ParseClassBody(ClassAST* ret)
 	};
 	AccessModifier access;
 	while (tokenizer.CurTok == tok_newline) tokenizer.GetNextToken();
-	std::unordered_set<string> anno;
+	std::vector<string> anno;
+	std::unordered_map<string, int> anno_occurrence;
```

Comment:
Why not just use a bool variable to mark if there is a "pure_virtual"? I think the map is unnecessary.